drydock_provisioner/drivers/node/maasdriver/driver.py

```diff
@@ -16,8 +16,10 @@
 import traceback
 import sys
 
+from oslo_config import cfg
```

Comment:
You're importing oslo_config but I don't think you're using it. You're using the config directly from drydock_provisioner instead, which should already be registered with oslo_config.